frontend/packages/console-app/src/components/nodes/NodeDetailsPage.tsx

```diff
@@ -34,8 +34,8 @@ const NodeDetailsPage: React.FC<React.ComponentProps<typeof DetailsPage>> = (pro
       ...(_.find(
         node?.metadata?.labels,
         (label) =>
-          label['corev1.LabelOSStable'] !== 'windows' ||
-          label['node.openshift.io/os_id'] !== 'Windows',
+          (label['corev1.LabelOSStable'] && label['corev1.LabelOSStable'] !== 'windows') ||
+          (label['node.openshift.io/os_id'] && label['node.openshift.io/os_id'] !== 'Windows'),
       )
```

Comment:
This is simpler as

```ts
!_.some(node?.metadata?.labels, (label) => {
  return label['corev1.LabelOSStable'] === 'windows' || label['node.openshift.io/os_id'] === 'Windows';
})
```

Reply:
Since the `labels` is not a nested object the iterated `label` will be the just the value of label. Needed to update it to:
```js
!_.some(node?.metadata?.labels, (v, k) => (k === 'node.openshift.io/os_id' && v === 'Windows') || (k === 'corev1.LabelOSStable' && v === 'windows'))
```